CONTRIBUTING.md

```diff
@@ -29,7 +29,10 @@ If you'd like to contribute code, here are some things to keep in mind.
   backwards-compatible with existing installations. If you must change something
   in a non-backwards-compatible way, document it in your pull request description and
   tag it with "Caveat For Existing Projects" so that we know that the change needs
-  to be mentioned in release notes.
+  to be mentioned in release notes. Non-backwards-compatible changes should be sent
+  to the next major release(eg. 16.X -> 17.0) while backwards-compatible changes should
+  be sent to the next minor release(eg. 16.1.0 -> 16.2.0 or 16.1.0 -> 16.1.1 depending
+  on how minor the changes are)
```

Comment:
I'd change the wording of this to say backwards compatible new features should go to 16.1.0 -> 16.2.0 and backwards compatible bug fixes should go 16.1.0 -> 16.1.1.

I'd also avoid "real" LORIS version numbers to avoid needing to update this for every release.